pkg/cmd/cli/backup/create.go

```diff
@@ -295,7 +295,7 @@ func (o *CreateOptions) Run(c *cobra.Command, f client.Factory) error {
 // Resource names in the list are in format 'namespace/resourcename' and separated by commas.
 // Key-value pairs in the mapping are separated by semi-colon.
 // Ex: 'pods=ns1/pod1,ns1/pod2;persistentvolumeclaims=ns1/pvc4,ns1/pvc8'.
-func parseOrderedResources(orderMapStr string) (map[string]string, error) {
+func ParseOrderedResources(orderMapStr string) (map[string]string, error) {
```

Comment:
The name of the func in the comment should be updated accordingly

Reply:
Oh, I missed that, thanks.
Have changed the name there now too.

Reply:
thanks!